base/linalg/rowvector.jl

```diff
@@ -83,6 +83,7 @@ julia> transpose(v)
 @inline adjoint(rowvec::RowVector{<:Real}) = rowvec.vec
 
 parent(rowvec::RowVector) = rowvec.vec
+Base.vec(rowvec::RowVector) = rowvec.vec
```

Comment:
No other method in this file uses `Base.`.

Reply:
No, but they probably should. It is a mess to `import` something in some other file and extend it here.

Reply:
Probably, but then better fix all occurrences or none.